lambda/package.json

```diff
@@ -4,14 +4,14 @@
     "description": "Fruit Shop Demonstration Skill",
     "main": "index.js",
     "scripts": {
-        "test": "node_modules/mocha/bin/_mocha \"test/test.js\"",
+        "test": "node_modules/mocha/bin/_mocha test/test.js",
```

Comment:
Nit: Can we also add targets for building the interaction model and also add a watch mode around that?

Reply:
build_interaction_model step is provided as part of launch.json under ide/vscode.
watch mode is usually for typescript, this a javascript skill sample :)

Reply:
aah.. my bad.. one more reason to have a typescript sample 😄